templates/components/engines/typescript/agent/chat.ts

```diff
@@ -28,7 +59,21 @@ export async function createChatEngine() {
     const config = JSON.parse(
       await fs.readFile(path.join("config", "tools.json"), "utf8"),
     );
-    tools = tools.concat(await ToolsFactory.createTools(config));
+
```

Comment:
how can we use the `ToolsFactory` to create tools that are in the templates/components/engines/typescript/agent/tools folder?